app/views/content/salaries-and-benefits.md

```diff
@@ -56,57 +56,52 @@ keywords:
 
 $new-salaries$
 
-For helping to shape the next generation, you’re entitled to a competitive salary, generous holidays, and a substantial pension.
+If you’re a primary or secondary teacher in England, your salary will depend on the type of school you work in, where the school is, and the pay range you’re on. 
 
-These salaries apply to maintained primary, secondary and special schools in England (schools funded by local authorities).
+These pay ranges apply to maintained schools (schools funded by local authorities). 
```

Comment:
@EllieNodder Is it clear enough that `These` refers to the pay ranges below rather than the mention in the preceding sentence?

Would maybe suggest putting '`The pay ranges below apply to` and then changing the following sentence to `often follow these pay ranges.`?